content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,236 @@
+Once the *Accounting* app has been installed, foreign currencies can be added to the database.
+Navigate to :menuselection:`Accounting app --> Configuration --> Settings`, scroll to the
+:guilabel:`Currencies` section, and locate the :guilabel:`Main Currency` setting.
 
-* Create one pricelist per currency. A new *Currency* field shows up in pricelist setup form.
+.. image:: currencies/main-currency-setting-page.png
+   :align: center
+   :alt: How the main currency feature appears on settings page in Odoo Accounting.
+
+Odoo automatically sets the main currency as the currency of the country the company is based in.
+
+To change the main currency of the company, select the drop-down menu in the :guilabel:`Currency`
+field, select the desired currency, and be sure to :guilabel:`Save` the changes.
 
 .. tip::
-    To activate a new currency, go to :menuselection:`Accounting --> Configuration --> Currencies`,
-    select it in the list and press *Activate* in the top-right corner. Now it will show up in
-    currencies drop-down lists.
+   To ensure currency rates are updated automatically, enable the *automatic currency rates* feature
+   on the *Accounting* settings page (:menuselection:`Accounting app --> Configuration --> Settings
+   --> Currencies section`).
+
+   .. image:: currencies/automatic-currency-rates.png
+      :align: center
+      :alt: How the main currency feature appears on settings page in Odoo Accounting.
 
-Prices in foreign currencies can be defined in two fashions.
+   Click the checkbox beside the :guilabel:`Automatic Currency Rates` feature, choose a designated
+   bank to get the currency rates from in the :guilabel:`Service` field drop-down menu, and select
+   an :guilabel:`Interval` of time for the updates to take place. Then determine when the date of
+   the :guilabel:`Next Run` should be.
 
-Automatic conversion from public price
-======================================
+   To instantly update the currency rates, click the :guilabel:`🔁 (circular arrows)` icon, located
+   to the right of the :guilabel:`Next Run` field.
 
-The public price is in your company's main currency (see
-:menuselection:`Accounting --> Settings`) and is set in product detail form.
+   When all configurations are complete, be sure to :guilabel:`Save` all changes.
 
-.. image:: currencies/public_price.png
+.. note::
+   All payment methods **must** be in the same currency as the sales journal, or the company
+   currency, if the company currency is not set. If it is not the same, a :guilabel:`Validation
+   Error` message appears.
+
+View, edit, and add currencies
+==============================
+
+To view, edit, and add currencies to the database, making them available on pricelists and on the
+:guilabel:`Main Currency` drop-down menu, click the :guilabel:`Currencies` link, located beneath the
+:guilabel:`Currency` field on the *Accounting* app's :guilabel:`Settings` page.
```

Comment:
Since the user is being directed to the Accounting app's settings page, we can use menuselection
```suggestion
:guilabel:`Currency` field on the :menuselection:`Accounting app --> Settings` page.
```